textstat/textstat.py

```diff
@@ -72,6 +67,7 @@ def get_grade_suffix(grade):
 class textstatistics:
     __lang = "en_US"
     text_encoding = "utf-8"
+    round_outputs = True
```

Comment:
As we're adding the ability to turn off rounding, it might also be nice to add the ability to set the rounding amount.

```python
>>> textstat.round_points = 6
>>> textstat.avg_letter_per_word(text)
3.637483
```

I'm not sure what the implementation would look like, maybe:

```python
round_points = None


def _legacy_round(points=0):
    points = self.round_points if self.round_points else points
```

I'll leave this up to you though. If you want to implement it, 👍. If not, no worries.